portal-ui/src/screens/Console/HealthInfo/HealthInfo.tsx

```diff
@@ -54,6 +56,10 @@ const styles = (theme: Theme) =>
       border: "1px solid #EAEDEE",
       borderRadius: 4,
     },
+    loading: {
+      paddingTop: "8px",
+      paddingLeft: "40px",
```

Comment:
Please remove unit as this is using pixels as default:

```suggestion
      paddingTop: 8,
      paddingLeft: 40,
```

Reply:
done